torch/fx/proxy.py

```diff
@@ -51,7 +51,14 @@ def create_arg(self, a: Any) -> Argument:
         """
         # aggregates
         if isinstance(a, (tuple, list)):
-            return type(a)(self.create_arg(elem) for elem in a)
+            if hasattr(a, '_fields'):
```

Comment:
Seems fine, but I'd split the name tuple isinstance check to the top level to make it more clear.
`if isinstance(x, named_tuple):`